composer.json

```diff
@@ -148,5 +148,13 @@
         "branch-alias": {
             "dev-master": "1.0-dev"
         }
+    },
+    "scripts": {
+        "check-style": [
+            "php-cs-fixer fix --config-file=.php_cs",
+            "php-formatter f:h:f src",
+            "php-formatter f:u:s src"
+        ],
+        "test": "phpunit"
```

Comment:
No because `bin/` files are executed first.

https://getcomposer.org/doc/articles/scripts.md#writing-custom-commands :
> Note: Composer's bin-dir is pushed on top of the PATH so that binaries of dependencies are easily accessible as CLI commands when writing scripts.